acceptance/acceptance_test.go

```diff
@@ -678,44 +678,59 @@ func testAcceptance(
 						t.Log("app is runnable")
 						assertMockAppRunsWithOutput(t, repoName, "Launch Dep Contents", "Cached Dep Contents")
 
+						version, err := packVersion(packPath)
+						h.AssertNil(t, err)
+						packSemver = semver.MustParse(strings.TrimPrefix(strings.Split(version, " ")[0], "v"))
+						packSupport := packSemver.GreaterThan(semver.MustParse("0.10.0")) || packSemver.Equal(semver.MustParse("0.0.0"))
```

Comment:
Could we change this variable name to `packSupportsCreator`? We have a function called `packSupports` that is called with the name of a command, so this would read similarly.

Reply:
Good idea! Thanks!